### PR TITLE
[ci skip][WIP] Freecad: update to 0.18

### PR DESCRIPTION
**Pull request by karl-nilsson**
ping @yopito

### CODE REVIEW

File: srcpkgs/libpyside2-devel [hunk 1]
```diff
@@ -0,0 +1 @@
+python3-pyside2/
```
Comment (Cogitri) [inline srcpkgs/libpyside2-devel]: No trailing slash here and in the other subpkgs please

File: srcpkgs/pyside2-tools/template [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+wrksrc="${_pkgname}"
+configure_args="-DUSE_PYTHON_VERSION=3
+ -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF"
+pycompile_module="pyside2uic"
+hostmakedepends="cmake"
+makedepends="python3-devel libshiboken2-python3-devel qt5-devel libpyside2-python3-devel"
+depends="python3"
+short_desc="LGPL-licensed Python3 bindings for the Qt5 toolkit"
+maintainer="yopito <pierre.bourgin@free.fr>"
+license="GPL-3.0-or-later"
```
Comment (Cogitri) [inline srcpkgs/pyside2-tools/template]: The license conflicts with the short desc

File: srcpkgs/pyside2-tools/template [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Template file for 'pyside2-tools'
+pkgname=pyside2-tools
+version=5.12.2
+revision=1
+_pkgname="pyside-setup-everywhere-src-${version}"
+wrksrc="${_pkgname}"
+configure_args="-DUSE_PYTHON_VERSION=3
+ -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=OFF"
```
Comment (Cogitri) [inline srcpkgs/pyside2-tools/template]: Do we really need to specify the BUILD_TYPE here?

File: srcpkgs/python3-pivy/template [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+license="0BSD"
+homepage="https://bitbucket.org/Coin3D/pivy"
+distfiles="https://bitbucket.org/Coin3D/pivy/get/${version}.tar.gz"
+checksum=2ee684dbf0f52c05b491c5423013fd82e679614bffee23641485248c8a790c4e
+nocross="use coin-config script"
+
+post_extract() (
+	# use setup_old.py until have Coin with cmake support
+	mv setup.py setup_new.py
+	mv setup_old.py setup.py
```
Comment (Cogitri) [inline srcpkgs/python3-pivy/template]: Just mv setup_old.py to setup.py, we don't need the new one yet, do we?

**Comment by yopito** [comment]
IMHO, some doubts about this PR:

* FreeCAD 0.18 is not yet released.  
  upstream project is still actively working on it (see https://forum.freecadweb.org/viewtopic.php?f=10&t=30340)
* not sure that Pyhton for Qt stack (shiboken, etc) version 5.12 has a good support of Qt 5.11
* I have some WIP on my own that is just waiting 0.18 get released, that is close to this PR, but also with some significant cleanup .. See https://github.com/yopito/void-packages/tree/freecad.current
* It's strange to see new packages where the committer is not is not the maintainer ;)

**Comment by karl-nilsson** [comment]
@yopito 

- v0.18 builds have been uploaded to the [freecad stable PPA](https://launchpad.net/~freecad-maintainers/+archive/ubuntu/freecad-stable)
- the github releases page has an entry for [0.18](https://github.com/FreeCAD/FreeCAD/releases/tag/0.18)
- I'll close this and wait for your PR

**Comment by yopito** [comment]
@karl-nilsson: OK, sorry for the situation. I confirm that FreeCAD 0.18 is not yet released

**Pull request closed without merging**